toolkit/tools/pkg/imagecustomizerlib/resolvedconfig.go

```diff
@@ -57,6 +57,9 @@ type ResolvedConfig struct {
 	// UKI
 	Uki *imagecustomizerapi.Uki
 
+	// COSI compression settings
+	CosiCompression *imagecustomizerapi.CosiCompression
```

Comment:
Use `imagecustomizerapi.CosiCompression` instead of `*imagecustomizerapi.CosiCompression` since we only care if the sub fields are provided or not.